hw/top_earlgrey/rdc/chip_earlgrey_asic_scenario.tcl

```diff
@@ -74,6 +74,13 @@ set_reset_scenario { \
     {constraint {@t0 0}}} \
   } -name SpidRstTxFifo -comment "SPI_DEVICE Async TX FIFO Functional Reset"
 
+# SPI_DEVICE CSb Reset. IP reset should be stable
```

Comment:
ah i see, so as i understand it, this reset scenario is when the input software / aon controls are all static, and only csb is toggled, is that right?

Reply:
yep. If that assumption is not correct, let me know. I will revise the scenario to toggle both (will create a ton of violations though..)

Reply:
i think that's correct for the normal functional case.
But let's say we "do" need to reset the IP because the host caused us to get stuck (for whatever reason), what would be the right procedure to do it?

Reply:
That's good Q. One of the scenario should cover that as I do not specify any CSb constraints while SW issuing SW_RST_REQ. The scenario is `RstMgrSwRst`.

The scenario reports 22 violations in SPI_DEVICE right now. I should review them. :)

Reply:
sounds good! But on that note though, did we add the thing where we either silenced csb or clock when the device is in reset?
Because if we did something like that, then technically we don't really have an issue right? Because the reset would just keep the interface completely quiescent.  Anyways, that's not related this PR, so feel free to merge :)

Reply:
I tried to, but somehow, if I specify that restriction, it causes errors as two reset scenarios are in conflict. (SwRst silences CSb, CSb scenario silences SwRst). I could not solve that issue yet.